azure-kusto-data/azure/kusto/data/_models.py

```diff
@@ -16,6 +18,18 @@
 except ImportError as e:
     HAS_PANDAS = False
 
+aio_installed = True
+try:
+    from aiohttp import ClientResponse, ClientSession
+except ImportError:
+    aio_installed = False
+
+
+def _get_dynamic_response_type():
```

Comment:
where do you use this?

plus, not sure why you need this code here, everything aio is under subfolder